### DISCUSSION TITLE
Unable to open due to original signed code

### DISCUSSION THREAD

**Discussion by chris-v88** [asb2m10/dexed#321] [Q&A]
Heyy, so I have this notif when I open this installer. I'm on Big Sur Mac, M1. Please let me know when this is fixed.

![image](https://user-images.githubusercontent.com/20401958/147766732-e90212f9-6e9e-49fa-9126-9b11081927ae.png)

**Comment by asb2m10** [comment]
Sorry about that, I don't have a Apple Dev ID yet to sign the package. 

What you can do is once you see that message, select "Top Apple" - "System Preference" - "Security & Privacy" and there you should see "open dexed-0.9.6.mpkg anyway".

Let me know if this works.
